Mathlib/Combinatorics/SimpleGraph/Matching.lean

```diff
@@ -442,6 +442,8 @@ lemma IsCycles.snd_of_mem_support_of_isPath_of_adj [Finite V] {v w w' : V}
   rw [Subgraph.adj_comm, Subgraph.adj_iff_of_neighborSet_equiv e (Set.toFinite _)]
   exact hadj.symm
 
+-- `Fintype V` is necessary for use in the termination measure, `Finite V` does not suffice.
+set_option linter.unusedFintypeInType false in
 private lemma IsCycles.reachable_sdiff_toSubgraph_spanningCoe_aux [Fintype V] {v w : V}
```

Comment:
You can change the `termination_by` to use `Nat.card` instead of `Fintype.card`, then adding a `have := Fintype.ofFinite V` to `decreasing_by` works